templates/en/objection-default.txt

```diff
@@ -0,0 +1,15 @@
+To Whom It May Concern:
+
+I am hereby objecting to the processing of personal data concerning me for direct marketing purposes according to Article 21(2) GDPR. This objection includes the use for profiling to the extent that it is related to direct marketing.
```

Comment:
Well, 21(2) doesn't.

And while, 21(1) is a lot broader in the scope of the processed data, it applies to significantly fewer people. The data subject has to justify the objection based on their particular situation. Examples I have heard of here include witness protection programs and mothers' shelters. This is not relevant for most people.

Case in point: The Topf Secret templates by FragDenStaat initially included a blanket objection according to Art. 21(1) GDPR which they have since removed as it doesn't apply.

Also see the previous discussion in datenanfragen/website#65.